tests/ui/traits/const-traits/const-bounds-non-const-trait.stderr

```diff
@@ -2,21 +2,35 @@ error: `~const` can only be applied to `#[const_trait]` traits
   --> $DIR/const-bounds-non-const-trait.rs:6:21
    |
 LL | const fn perform<T: ~const NonConst>() {}
-   |                     ^^^^^^
+   |                     ^^^^^^ can't be applied to `NonConst`
+   |
+help: mark `NonConst` as `const`
```

Comment:
this help isn't what is actually being suggested

"mark `NonConst` as `#[const_trait]` to allow it to have const implementations"

Also, make sure that this suggestion only fires on nightly? Or better yet, when `const_trait_impl` is enabled.